__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index.js");
+
+afterAll(() => {
+  if (db.end) db.end();
+});
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("GET api/topics", () => {
+  test('"status:200, responds with an array of topic objects" ', () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toBeInstanceOf(Array);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
```

Comment:
v nice testing - would even check length of topics to make sure all the data you expect is in there